cpp/arcticdb/toolbox/python_bindings.cpp

```diff
@@ -0,0 +1,56 @@
+#include <pybind11/functional.h>
+
+#include <arcticdb/storage/library.hpp>
+#include <arcticdb/storage/s3/s3_storage_tool.hpp>
+#include <arcticdb/toolbox/library_tool.hpp>
+#include <arcticdb/util/memory_tracing.hpp>
+
+
+namespace arcticdb::toolbox::apy {
+
+void register_bindings(py::module &m) {
+    auto tools = m.def_submodule("tools", "Library management tool hooks");
+    using namespace arcticdb::toolbox::apy;
+    using namespace arcticdb::storage;
+
+    tools.def("print_mem_usage", &util::print_total_mem_usage);
+
+    py::class_<LibraryTool, std::shared_ptr<LibraryTool>>(tools, "LibraryTool")
+            .def(py::init<>([](std::shared_ptr<Library> lib) {
+                return std::make_shared<LibraryTool>(lib);
+            }))
+            .def("read", &LibraryTool::read)
```

Comment:
I love the cleanup. Worth adding a docstring just to this method to make it clear what usage looks like? This is a super useful method IMO.

Reply:
Oh I see, this is probably not the right place for a docstring. Should be in the Python wrapper.